inst/examples/census/config.yml

```diff
@@ -24,9 +23,8 @@ cloudml:
   eval_file                : "gs://rstudio-cloudml-demo-ml/census/data/adult.test"
   train_file               : "gs://rstudio-cloudml-demo-ml/census/data/adult.data"
 
-  # Outputs (to run within another gcloud account, substitute your own buckets)
-  job_dir                  : !expr cloudml::unique_job_dir("gs://rstudio-cloudml-demo-ml/census/jobs")
-  staging_bucket           : "gs://rstudio-cloudml-demo-ml"
```

Comment:
I think that should be fine; it just implies that things will be staged within the `job_dir`. I think this is actually nicer as right now every deployment clogs the root bucket directory up with a new directory.